crawler/config/centres.py

```diff
@@ -287,7 +281,7 @@ def get_centres_config(config: Config, data_source: str = "") -> List[CentreConf
         if not centres_collection_exists:
             # Populate the centres collection from the config values
             create_index(centres_collection, FIELD_CENTRE_NAME, unique=True)
-            populate_collection(centres_collection, config.CENTRES, FIELD_CENTRE_NAME)  # type: ignore
+            populate_mongo_collection(centres_collection, config.CENTRES, FIELD_CENTRE_NAME)  # type: ignore
```

Comment:
If this is creating all new data, why is it doing 'upsert'?

Reply:
That's the existing workflow.  I agree it doesn't have to be upsert but in this case they're equivalent because we check for the collection before performing the operation.